README.md

```diff
@@ -122,20 +137,20 @@ If you plan to develop with TorchServe and change some of the source code, you m
 1. Make your changes executable
 
     ```bash
-    pip install -e .
+    pip install .
```

Comment:
The instruction is to make it executable, but you remove the executable flag.

Reply:
-e flag is for "editable" and not "executable", which is causing the problem when we install from source. This flag skips the gradle build and hence the frontend jar is not generated leading to "ClassNotFoundException" while starting `torchserve`

Reference : https://pip.pypa.io/en/stable/reference/pip_install/#options